### PR TITLE
Update readme to note this repo is archived

### PR DESCRIPTION
**Pull request by stevenday**
As discussed on our sprint call yesterday, we think we can archive this repo and use OCDSKit in the future (at least until we find a reason we can't, at which point we'll reconsider).

I've made this PR as a way of flagging this to @ScatteredInk and anyone else interested. Speak now or forever hold your peace! 

See also: https://github.com/openownership/bods-dev-handbook/pull/16

### CODE REVIEW

**Comment by stevenday** [comment]
@odscjames or @Bjwebb - FYI - I think I might need adding to the readthedocs project for this repo to put this live, if deployment isn't automatic? Alternatively, if one of you could deploy it once merged, that would be much appreciated.

**Comment by jpmckinney** [comment]
I’m happy to merge into OCDS Kit tooling related to the schema files of BODS (as they follow the same format as OCDS). However, tooling related to data conforming to BODS deserves its own repo. Should the message be clarified to say “the schema files of the Beneficial ...”?

**Comment by stevenday** [comment]
Thanks for the clarification @jpmckinney and sorry for any confusion. I've updated the text to make that clearer - as we don't have any tools for working with the data at the moment, we'll leave this repo archived until we do.

**Review by ScatteredInk — APPROVED** [review]
Thanks @stevenday - looks good to me.

**Comment by Bjwebb** [comment]
@stevenday I've added you as Maintainer, but it should go live when merged into master.

**Pull request merged**
